src/analytics/store-analytics.js

```diff
@@ -74,43 +73,45 @@ const askAnalysis = (startDate, endDate, cnpjs, domain) => {
       ${yearMonthFilter}
     group by
       st.cnpj, s.domain, s.documentStatus
+    order by
+      st.cnpj, s.domain
     limit
-      ${cnpjs.length}
+      2000
   `;
 
   console.log(stmt);
 
   return prestoClient
     .sendStatement(stmt)
-		.then((result) => {
-			console.log(result)
-
-			const allAnaly = []
-			let i, j
-			for (i = 0; i < result.data.length; i++) {
-				const cnpjAnaly = {}
-				for (j = 0; j < result.data[i].length; j++) {
-					cnpjAnaly[result.columns[j].name] = result.data[i][j]
-				}
-				allAnaly.push(cnpjAnaly)
-			}
-			console.log(allAnaly)
+    .then((result) => {
+      console.log(result)
+
+      const allAnaly = []
+      let i, j
+      for (i = 0; i < result.data.length; i++) {
+        const cnpjAnaly = {}
+        for (j = 0; j < result.data[i].length; j++) {
+          cnpjAnaly[result.columns[j].name] = result.data[i][j]
+        }
+        allAnaly.push(cnpjAnaly)
```

Comment:
Some places you use `;`, others don't. Please, keep one style.

Reply:
If doing right, use eslint already in this project